pkg/tokenreviews/tokenvalidation/validators/timeoutvalidator_test.go

```diff
@@ -0,0 +1,272 @@
+package validators
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/clock"
+
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	userv1 "github.com/openshift/api/user/v1"
+	oauthfake "github.com/openshift/client-go/oauth/clientset/versioned/fake"
+	oauthclient "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+)
+
+func TestAuthenticateTokenTimeout(t *testing.T) {
```

Comment:
aren't there also just copied from somewhere?

Reply:
Those needed to be split and fixed so that they each tested the unit they were supposed to test: https://github.com/openshift/oauth-apiserver/pull/9#discussion_r568620634